src/Manager/ProcessManager.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Synolia\SyliusAkeneoPlugin\Manager;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Process\Process;
+
+class ProcessManager implements ProcessManagerInterface
+{
+    private array $processes = [];
+
+    private bool $instantProcessing = false;
+
+    public function __construct(
+        protected \BluePsyduck\SymfonyProcessManager\ProcessManager $processManager,
+        protected LoggerInterface $logger,
```

Comment:
```suggestion
        protected LoggerInterface $akeneoLogger,
```

We create an akeneoLogger, but as i see, it's not always used. Do we neet to refactor that ?